jwql/website/apps/jwql/templates/not_authenticated.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block preamble %}
+
+    <title>About - JWQL</title>
```

Comment:
Tiny change - this page should probably have an accurate name.

```suggestion
    <title>Not Authenticated - JWQL</title>
```

Reply:
@laurenmarietta Good catch!